indexer-events/migrations/20230120134210_add_partitions.sql

```diff
@@ -0,0 +1,121 @@
+-- 16250976000000000000000000000000000 (2021-07-01)
+-- 16277760000000000000000000000000000 (2021-08-01)
+-- 16304544000000000000000000000000000 (2021-09-01)
+-- 16330464000000000000000000000000000 (2021-10-01) 
+-- 16357248000000000000000000000000000 (2021-11-01) 
+-- 16383168000000000000000000000000000 (2021-12-01)
+
+-- 16409952000000000000000000000000000 (2022-01-01)
+-- 16436736000000000000000000000000000 (2022-02-01) 
+-- 16460928000000000000000000000000000 (2022-03-01) 
+-- 16487712000000000000000000000000000 (2022-04-01) 
+-- 16513632000000000000000000000000000 (2022-05-01) 
+-- 16540416000000000000000000000000000 (2022-06-01) 
+-- 16566336000000000000000000000000000 (2022-07-01)
+-- 16593120000000000000000000000000000 (2022-08-01)
+-- 16619904000000000000000000000000000 (2022-09-01)
+-- 16645824000000000000000000000000000 (2022-10-01) 
+-- 16672608000000000000000000000000000 (2022-11-01) 
+-- 16698528000000000000000000000000000 (2022-12-01) 
+
+-- 16094592000000000000000000000000000 (2021-01-01 to 2021-01-01) - OK (Old)
+-- 16725312000000000000000000000000000 (2023-01-01 to 2023-02-01) - OK
+-- 16752096000000000000000000000000000 (2023-02-0 to 2023-03-01) - OK
+-- 16776288000000000000000000000000000 (2023-03-01 to 2023-04-01) 
+
+-- NOTE: Team suggested to use a new table name fungible_token_events instead of coin_events
+
+BEGIN TRANSACTION;
+-- Rename table and indexes to <table/index>_old
+ALTER TABLE coin_events RENAME TO fungible_token_events_old;
+
+ALTER TABLE coin_events_affected_account_id_idx RENAME TO fungible_token_events_affected_account_id_idx_old;
+ALTER TABLE coin_events_block_height_idx RENAME TO fungible_token_events_block_height_idx_old;
+ALTER TABLE coin_events_receipt_id_idx RENAME TO fungible_token_events_receipt_id_idx_old;
+ALTER TABLE coin_events_block_timestamp_idx RENAME TO fungible_token_events_block_timestamp_idx_old;
+
+-- Re-Create the table with partition config
+CREATE TABLE fungible_token_events (
+	event_index numeric(38) NOT NULL,
+	standard text NOT NULL,
+	receipt_id text NOT NULL,
+	block_height numeric(20) NOT NULL,
+	block_timestamp numeric(20) NOT NULL,
+	contract_account_id text NOT NULL,
+	affected_account_id text NOT NULL,
+	involved_account_id text NULL,
+	delta_amount numeric(40) NOT NULL,
+	cause text NOT NULL,
+	status text NOT NULL,
+	event_memo text NULL,
+	CONSTRAINT fungible_token_events_pkey PRIMARY KEY (event_index)
+)
+PARTITION BY RANGE (event_index);
+
+-- Re-Create indexes for the new table
+CREATE INDEX fungible_token_events_affected_account_id_idx ON public.fungible_token_events USING btree (affected_account_id);
+CREATE INDEX fungible_token_events_block_height_idx ON public.fungible_token_events USING btree (block_height);
+CREATE INDEX fungible_token_events_receipt_id_idx ON public.fungible_token_events USING btree (receipt_id);
+CREATE INDEX fungible_token_events_block_timestamp_idx ON fungible_token_events USING btree (block_timestamp);
```

Comment:
nit: let's make it unified
```suggestion
CREATE INDEX fungible_token_events_block_timestamp_idx ON public.fungible_token_events USING btree (block_timestamp);
```

Reply:
thanks, will do